pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/filter/BaseFilter.java

```diff
@@ -32,6 +34,10 @@ public class BaseFilter {
     protected int from;
     protected int to;
     protected SortBy sortBy;
+    @Setter
+    @Getter
+    // whether the filter method should handle parsing returned objects into DTO
+    private boolean parseDtos = true;
```

Comment:
the same concern about additional boolean property, did not get it became necessary not to parse?

Reply:
Parsing took too long -- we only have 10 minutes to return data before AppEngine will time requests out.   Running profiling on the export code (when it was taking almost 5 minutes just to export 100 participants) showed that the vast majority of the time was taken in transforming the ES result to our Java object, and then parsing bits and pieces back to maps as we were reading unstructured Map<String, Object>s.  So I eliminated the back-and-forth parsing

Reply:
was it really parsing time? we use `jackson` library mainly to convert into Java object mainly, I understand your point but to sacrifice the design because of performance and make code messy I don't like this idea, maybe introduce child class of BaseFilter which will be focused on what you need instead of putting it directly here

Reply:
Totally agree -- everything would be better if we didn't have to worry about the performance.  :).  I've created an Unparsed subclass as you suggest to better isolate this code